### PR TITLE
recipe for mosestokenizer

### PR DESCRIPTION
**Pull request by monaldoj**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages
- [ ] Build number is 0
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    JSON <http://json.org> encoder and decoder for Python 2.5+ and
+    Python 3.3+. It is pure Python code with no dependencies, but includes
+    an optional C extension for a serious speed boost.
+  doc_url: http://simplejson.readthedocs.io/
+  dev_url: https://github.com/simplejson/simplejson
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
+    # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
```
Comment (marcelotrevisani) [inline recipes/example/meta.yaml]: Please, remove the comments

Comment (marcelotrevisani) [reply recipes/example/meta.yaml]: Please, revert it

File: recipes/mosestokenizer/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - mosestokenizer
+  commands:
+    - moses-tokenizer --help
+    - moses-detokenizer --help
+    - moses-punct-normalizer --help
+    - moses-sent-splitter --help
+
+about:
+  home: https://bitbucket.org/luismsgomes/mosestokenizer
+  license: GNU Lesser General Public v2 or later (LGPLv2+)
```
Comment (marcelotrevisani) [inline recipes/mosestokenizer/meta.yaml]: Please, change it to 
```
license: GPL-2.0
```

File: recipes/toolwrapper/meta.yaml [hunk 3]
```diff
@@ -1,41 +0,0 @@
-{% set name = "toolwrapper" %}
```
Comment (marcelotrevisani) [inline recipes/toolwrapper/meta.yaml]: Please, do not touch others recipes, revert it please

File: recipes/mosestokenizer/meta.yaml (hunk 2 above)
Comment (monaldoj) [reply recipes/mosestokenizer/meta.yaml]: changed in new commit

File: recipes/example/meta.yaml (hunk 1 above)
Comment (monaldoj) [reply recipes/example/meta.yaml]: the example/meta.yaml should be untouched like you recommend, i'm not sure where you're seeing differences

File: recipes/toolwrapper/meta.yaml (hunk 3 above)
Comment (monaldoj) [reply recipes/toolwrapper/meta.yaml]: similar to the first concern with the changed `example/meta.yaml` this `toolwrapper/meta.yaml` doesn't exist in the current commit

Comment (monaldoj) [reply recipes/toolwrapper/meta.yaml]: unless i'm missing something obvious (which i very well could be)

**Pull request merged**